index.html

```diff
@@ -0,0 +1,64 @@
+<html>
+<head>
+  <title>The Lorem Micro Blog</title>
+  <link rel="stylesheet" type="text/css" href="styles.css"/>
+</head>
+<body>
+  <nav>
+    <h1>
+      The Lorem Micro Blog
```

Comment:
Yes, I used "nav" is for the yellowish bar fixed to the top of the screen, and "header" for the green area lower on the page.